components/alertbanner/stories/alertbanner.test.js

```diff
@@ -1,38 +1,8 @@
-import { Container, Variants } from "@spectrum-css/preview/decorators";
+import { Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
-/**
- * Examples of options for contextual action button and close button.
- */
-export const AlertBannerVariantsTemplate = (args, context) => Container({
-	withBorder: false,
-	direction: "column",
-	containerStyles: { "align-items": "stretch" },
-	wrapperStyles: { "align-items": "stretch" },
-	content: [
-		Template({
-			...args,
-			variant: "neutral",
-			text: "Alert banner message. Use a short phrase to describe what's happening.",
-			actionButtonText: "Action",
-		}, context),
-		Template({
-			...args,
-			variant: "info",
-			text: "Alert banner message. Use a short phrase to describe what's happening.",
-			actionButtonText: "Action",
-		}, context),
-		Template({
-			...args,
-			variant: "negative",
-			text: "Alert banner message. Use a short phrase to describe what's happening.",
-			actionButtonText: "Action",
-		}, context),
-	],
-}, context);
-
 export const AlertBannerGroup = Variants({
-	Template: AlertBannerVariantsTemplate,
+	Template: Template,
 	TestTemplate: Template,
```

Comment:
Because the `Template` that we're using on the story's page is now the same as the `TestTemplate`, I'm pretty sure we can delete the `TestTemplate` line. 

```suggestion
Template,
```

Reply:
Good catch! All done. ✨